webpack.config.js

```diff
@@ -31,22 +31,34 @@ module.exports = {
         loader: 'babel-loader',
       },
       {
-        test: /\.(s*)css$/,
+        test: /\.s?css$/,
         exclude: /node_modules/,
-        use: [
-          { loader: 'style-loader' },
+        // So foo.module.scss gets treated as a css module foo.css does not
```

Comment:
So it turns out that gatsby's webpack config checks for same thing I believe


![image](https://user-images.githubusercontent.com/142403/66449798-2783ce00-ea0b-11e9-8e3d-243a908dce8e.png)

and
E.g. in `cms/node_modules/gatsby/dist/utils/webpack-utils.js`:

![image](https://user-images.githubusercontent.com/142403/66449815-3a969e00-ea0b-11e9-91b1-1ff03f9d0f11.png)